.nvmrc

```diff
@@ -0,0 +1 @@
+20.15.0
```

Comment:
can this be locked to just major version or does it need to be 20.15.0?

Reply:
I just set it to the one we use in the Dockerfile - felt like it made most sense!